src/app/services/code-view/angular-code-service.ts

```diff
@@ -191,7 +193,13 @@ export class AngularCodeService extends CodeService {
         return function (this: JQuery.UrlAjaxSettings, data: any) {
             const files = data.files;
             codeService.replaceRelativeAssetsUrls(files, demosBaseUrl);
-            codeService.sharedFileContent = data;
+
+            if (util.isLocalhost && codeService.demosUrlsSize > 1){
```

Comment:
util.isLocalhost is redundant since this function will be used only under localhost